toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -628,6 +703,14 @@ func (b *LiveOSIsoBuilder) generateInitrdImage(rootfsSourceDir, artifactsSourceD
 		return fmt.Errorf("failed to initialize chroot object for %s:\n%w", rootfsSourceDir, err)
 	}
 
+	requiredRpms := []string{"squashfs-tools", "tar", "device-mapper"}
+	for _, requiredRpm := range requiredRpms {
+		logger.Log.Debugf("Checking if (%s) is installed", requiredRpm)
+		if !isPackageInstalled(chroot, requiredRpm) {
+			return fmt.Errorf("failed to find required package (%s). The following packages must be installed:%v", requiredRpm, requiredRpms)
```

Comment:
nit: `"package (%s) is not installed:\nthe following packages must be installed to generate an iso: %v"`

Reply:
pending

Reply:
fixed.